frontend/src/Components/RoadDetails/ImageClick.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+interface ImageClickProps {
+  imageUrl: string;
+  onClose: () => void;
+  onNavigate: (direction: number) => void;
+  currentImageIndex: number;
+  totalImages: number;
+}
+
+const ImageClick: React.FC<ImageClickProps> = ({ imageUrl, onClose }) => {
+  return (
+    <div className="image-click-modal">
+      <div className="image-modal-content">
+        <img src={imageUrl} alt="Zoomed Image" />
```

Comment:
```suggestion
        <img src={imageUrl} draggable="false" alt="Zoomed Image" />
```
So the user can't drag the image